spec/oga/xml/lexer/elements_spec.rb

```diff
@@ -307,4 +307,19 @@
       ]
     end
   end
+  
+  describe 'elements with inline dots' do
+    it 'lexes an element with inline dots' do
+        lex('<SOAP..TestMapping..MappablePerson env:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/" />').should == [
```

Comment:
This spec can be trimmed down to:

```ruby
it 'lexes an element with a dot in the name' do
  lex('<foo.bar />').should == [...]
end
```

The extra describe isn't really needed, nor is the attribute value. If attribute names are also allowed to contain dots this should be tested in a separate `it` block.